spec/factories/contact_messages.rb

```diff
@@ -0,0 +1,9 @@
+# encoding: UTF-8
+FactoryGirl.define do
+  factory :contact_message do |c|
```

Comment:
Unused block argument - `c`. You can omit the argument if you don't care about it.